packages/frontend/src/js/app/components/map/map.js

```diff
@@ -881,82 +890,75 @@ export default {
                 .classList[action]("mapPin--street");
         },
 
-        // Fonction de chargement des données geoJson régionales
+        // Fonction de chargement des marqueurs régionaux
         loadRegionalData() {
             this.regionalLayer.getLayers().forEach(layer => {
                 if (layer instanceof L.Marker) {
                     layer.remove();
                     return;
                 }
-
-                // Calcul de la position du marqueur
-                // On utilise la méthode pointOnFeature(), qui garantit que le point soit dans le polygone, plutôt que centroid()
-                const { feature } = layer;
-                const markerPosition = pointOnFeature(feature);
-                // Création du marqueur à partir de la long et lat retournées par pointOnFeature()
-                const lon = markerPosition.geometry.coordinates[0];
-                const lat = markerPosition.geometry.coordinates[1];
+            });
+            Object.keys(this.numberOfShantytownsBy.regions).forEach(key => {
                 const nbSites =
-                    this.numberOfShantytownsBy.regions[
-                        feature.properties.code
-                    ] || 0;
-                if (nbSites > 0) {
-                    this.circleWithText(
-                        this.map,
-                        [lat, lon],
-                        this.numberOfShantytownsBy.regions[
-                            feature.properties.code
-                        ] || 0,
-                        20,
-                        3,
-                        "region"
-                    ).addTo(this.regionalLayer);
-                    this.regionalLayer.setStyle({
-                        color: "white",
-                        weight: 8,
-                        opacity: 1
-                    });
-                }
+                    this.numberOfShantytownsBy.regions[key].sites > 0
+                        ? this.numberOfShantytownsBy.regions[key].sites
+                        : 0;
```

Comment:
La condition est bizarre je trouve ici.
Tu peux pas utiliser `this.numberOfShantytownsBy.regions[key].sites` directement ?